backend/utils/resource-utils.js

```diff
@@ -54,6 +54,22 @@ let options = {
   ]
 };
 
+let tagOnly = {
```

Comment:
Great job separating this out into another file and keeping the business logic in the utilities 👍